js/src/modal.js

```diff
@@ -108,20 +108,20 @@ class Modal extends BaseComponent {
       return
     }
 
-    if (this._isAnimated()) {
-      this._isTransitioning = true
-    }
-
     const showEvent = EventHandler.trigger(this._element, EVENT_SHOW, {
       relatedTarget
     })
 
-    if (this._isShown || showEvent.defaultPrevented) {
+    if (showEvent.defaultPrevented) {
       return
     }
 
     this._isShown = true
```

Comment:
Clearly for ordering reasons and only I would prefer 
```js 
this._isShown = true
```

to be after 
```js    
this._isTransitioning = true

```

Reply:
I did it so to use the same order as the hide() method. 😊